internal/internal_workflow_client.go

```diff
@@ -190,6 +202,9 @@ func (wc *WorkflowClient) ExecuteWorkflow(ctx context.Context, options StartWork
 // The current timeout resolution implementation is in seconds and uses math.Ceil(d.Seconds()) as the duration. But is
 // subjected to change in the future.
 func (wc *WorkflowClient) GetWorkflow(ctx context.Context, workflowID string, runID string) WorkflowRun {
+	// We intentionally don't "ensureIntialized" here because there is no direct
+	// error return path. Rather we let GetWorkflowHistory do it.
+
```

Comment:
I'd avoid words "we" and "I" in comments. Something more "dry" is better here:
```suggestion
	// "ensureIntialized" is not called here intentionally because there is no direct
	// error return path. GetWorkflowHistory will do it.

```